cli/commands/clean.js

```diff
@@ -89,6 +89,11 @@ exports.run = function (logger, config, cli) {
 	if (cli.argv.platforms) {
 		async.series(cli.argv.platforms.map(function (platform) {
 			return function (next) {
+				// scan platform SDK specific clean hooks
```

Comment:
Do we really need to look for a platform-specific cli/commands/_clean.js, load that and call scanHooks on it? Shouldn't we just call:
```javascript
cli.scanHooks(path.join(__dirname, '..', '..', platform, 'cli', 'hooks'));
```

Basically just say load all hooks from the platform specific cli hooks directory.